### DISCUSSION TITLE
Zen's userChrome.css to disable rounded corner

### DISCUSSION THREAD

**Discussion by gunir** [zen-browser/desktop#233] [Show and tell]
First, read [this guide](https://old.reddit.com/r/FirefoxCSS/wiki/index/tutorials#wiki_setup) but soon Zen'll add more CSS feature to things should become much easier (streamlined) so everyone can do this easily.

`userChrome.css`
```
:root:not([inDOMFullscreen="true"]) #tabbrowser-tabbox #tabbrowser-tabpanels .browserSidebarContainer {
  border-radius: unset!important;
  margin: unset!important;
}

:root:not([inDOMFullscreen="true"]) #tabbrowser-tabbox #tabbrowser-tabpanels[zen-split-view="true"] .browserSidebarContainer {
  margin: unset!important;
}
```

Result:
![image](https://github.com/user-attachments/assets/70298346-c2a4-4166-9d39-81cec4e792fd)

**Comment by mr-cheffy** [comment]
Thanks! Ill come back here once I have a theme market place!
